prime-angular-frontend/src/app/modules/adjudication/pages/health-authorities/health-auth-org-info-page/health-auth-org-info-page.component.ts

```diff
@@ -8,6 +8,8 @@ import { HealthAuthorityResource } from '@core/resources/health-authority-resour
 import { HealthAuthority } from '@shared/models/health-authority.model';
 
 import { AdjudicationRoutes } from '@adjudication/adjudication.routes';
+import { UtilsService } from '@core/services/utils.service';
+import { BaseDocument } from '@shared/components/document-upload/document-upload/document-upload.component';
```

Comment:
Can these lines be moved up?

Reply:
I think `import { UtilsService } from '@core/services/utils.service';` still needs to be moved up